eosdis-nasa/earthdata-pub-api
# EDPUB-1304: Fix auto add

# Description

Fix auto add so that only DAAC manager roles are auto added to submissions. Also add new non-initiator email template for request initialization for clarification.

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1304

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Push branch to SIT
3. Create a new submission.
4. Observe that the request initiator gets the request initialization email while DAAC managers get the new DAAC request initialization email.
5. Confirm that DAAC staff are no longer added to newly initialized requests.

---
